Library/Homebrew/formula.rb

```diff
@@ -1690,10 +1690,13 @@ def to_hash
     end
 
     hsh["requirements"] = requirements.map do |req|
+      name = (req.try(:comparator) == "<=" ? "maximum_" : "") + req.name
```

Comment:
Could this use an `if` and/or some intermediate variables? It's a little hard to follow as-is.